R/runSS.R

```diff
@@ -65,7 +65,7 @@ run_ss_model <- function(dir,
   if (os == "unix") {
     system(
       paste0(
-        "cd ", dir, ";", paste0(bin, " "),
+        "sudo chmod a+x ", bin, "; ", "cd ", dir, "; ", paste0(bin, " "),
```

Comment:
@e-perl-NOAA was this needed to guarantee that the permissions were set correctly?

Or could we change the permissions on the file within the repo only?

Reply:
I needed it to get the github action that runs the tests on the macos-14 runner to pass. However, I think to absolutely make sure the permissions are correct it wouldn't necessarily be a bad thing to have it in there, especially for macs. I feel like I've had to tell people often enough with macs to make sure the change the permission for the exe.

Reply:
yes, that's very true, I remember telling people that a lot as well. I think it's good to stay as-is then, thanks!